code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -184,3 +184,21 @@
 	else if(finished == STATION_VICTORY)
 		SSticker.mode_result = "loss - rev heads killed"
 		SSticker.news_report = REVS_LOSE
+
+//////////////////////////////////////////////
+//                                          //
+//           SYNDICATE TRAITORS             //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/latejoin/heretic_smuggler
+	name = "Heretic Smuggler"
+	antag_datum = /datum/antagonist/heretic
+	antag_flag = ROLE_CULTIST //TEMPORARY SOLUTION FIX BEFORE MERGING
+	protected_roles = list("Security Officer", "Warden", "Head of Personnel", "Detective", "Head of Security", "Captain")
```

Comment:
Prisoner

Reply:
Reason being, bureaucratic error can some times cause them to latejoin for instance.